symfony/plugins/orangehrmMarketPlacePlugin/modules/marketPlace/templates/ohrmAddonsSuccess.php

```diff
@@ -78,7 +78,7 @@
                             <div>
                                 <p><?php
                                     if (array_key_exists($addon['id'], $expirationDates)) {
-                                        echo "Expiration Date:- " . $expirationDates[$addon['id']];
+                                        echo __('Expiration Date:- ') . $expirationDates[$addon['id']];
```

Comment:
echo __('Expiration Date:- ') 

this should be corrected as follows. 
 echo __('Expiration Date').':- '; ........